darwin/DarwinProcess.c

```diff
@@ -6,6 +6,7 @@ in the source distribution for its full text.
 */
 
 #include "DarwinProcess.h"
+#include "Platform.h"
```

Comment:
Cf. styleguide on include order.

Reply:
I think I got it right this time, let me know.